### PR TITLE
add test1

### PR DESCRIPTION
**Pull request by julia951**

### CODE REVIEW

File: src/tests/sortingCheck.spec.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+
+test.describe('Test 1', () => {
+    test('Sort items on Inventory page', async ({ loginPage, inventoryPage, shopingCartPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin('standard_user', 'secret_sauce');
+
+       await inventoryPage.sortItems('az');
+        let itemNames = await inventoryPage.getItemNames();
```
Comment (hlazkov) [inline src/tests/sortingCheck.spec.js]: Please use const whereever its applicible

File: src/tests/sortingCheck.spec.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// test file (e.g., inventory.test.js)
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+
+test.describe('Test 1', () => {
+    test('Sort items on Inventory page', async ({ loginPage, inventoryPage, shopingCartPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin('standard_user', 'secret_sauce');
+
+       await inventoryPage.sortItems('az');
```
Comment (hlazkov) [inline src/tests/sortingCheck.spec.js]: Should we add other sorting options in this PR?

File: src/tests/sortingCheck.spec.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        let itemNames = await inventoryPage.getItemNames();
+        let sortedItemNames = [...itemNames].sort();
```
Comment (hlazkov) [inline src/tests/sortingCheck.spec.js]: What is the difference for [...itemNames].sort(); and itemNames.sort(); ?

File: src/tests/sortingCheck.spec.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        expect(itemNames).toEqual(sortedItemNames);
+
+       await inventoryPage.sortItems('za');
```
Comment (hlazkov) [inline src/tests/sortingCheck.spec.js]: Lets split to separate tests, so they would be independed

**Review by hlazkov — CHANGES_REQUESTED** [review]
Please pay attention to comments provided

**Review by hlazkov — CHANGES_REQUESTED** [review]
Lets add 2 tests for sorting by price as well!

**Review by EugenePetrik — APPROVED** [review]
Good job!

**Pull request merged**